src/languageSupport/languages/flux/lsp/connection.ts

```diff
@@ -155,8 +155,7 @@ class LspConnectionManager {
   }
 
   _editorChangeIsFromLsp(change) {
-    // heuristic. Not 100% accurate.
-    return !!change.forceMoveMarkers
```

Comment:
Made this change because of heuristic failing:
(see the 3rd change object).

<img width="606" alt="Screen Shot 2022-09-01 at 1 57 32 PM" src="https://user-images.githubusercontent.com/10232835/188026508-b9892bdc-1a84-447d-9677-9fe3a2c8dbed.png">